src/services/expenses.js

```diff
@@ -0,0 +1,92 @@
+'use strict';
+
+let expenses = [];
+
+const getMax = (array) => {
```

Comment:
Consider renaming 'getMax' to 'getNextId' since the function returns the next available ID for a new expense.